gf-guide/metadata.md

```diff
@@ -389,3 +369,62 @@ Example:
 ``` code
 minisite_url: "https://myfontwebsite.com"
 ```
+
+
+### Classification
+
+TO DO
+
+### Stroke
+
+TO DO
+
+### Sample text
+
+This key overrides the sample texts provided in the languages texprotos (that you can find in the [Lang](https://github.com/googlefonts/lang/tree/main/Lib/gflanguages/data/languages) repository).
+
+If sample text is given within a sample text group (e.g. poster, specimen), all fields within that group must be provided.
+
+- `masthead_full` is a 4 glyphs string (e.g. AaBb) used in a single-script masthead. It is used in specimen masthead component.
```

Comment:
What is the Specimen masthead component? Perhaps a picture could be of help here?

Reply:
We should take screens of the different groups yes